### DISCUSSION TITLE
About graphql endpoint

### DISCUSSION THREAD

**Discussion by waptik** [opencollective/opencollective#3816] [Q&A] [answered]
Hi!

Just a quick question, is an api key needed when querying your graphql endpoints?

Nice project BTW!

**Comment by SudharakaP** [comment, marked as answer]
@waptik : Thanks for your question. Yes an API key is needed. Read the following article on how to get the api and send requests and stuff; 

https://medium.com/open-collective/open-collective-graphql-api-preview-3b42ed1d55ff

And the following is the GraphQL schema documentation; 

https://graphql-docs-v2.opencollective.com/

Let us know if you need any further info. 😄

**Reply by SudharakaP** [reply]
To add some more info, you can also find the above information in the documentation; https://docs.opencollective.com/help/contributing/development/api

**Reply by waptik** [reply]
I'm glad to know that an API key needed.
I found a bug concerning this. I was about to file a bug report but wasn't sure about it.

**Reply by SudharakaP** [reply]
@waptik : You are welcome. ☺️

**Reply by waptik** [reply]
I just did that in #3817

**Reply by SudharakaP** [reply]
@waptik : Ops sorry, I somehow missed the part about the bug. Thanks for opening the issue. Will take a look. 🙃
